### PR TITLE
Casedb Fixtures

### PR DESCRIPTION
**Pull request by proteusvacuum**
Adds the ability to create a casedb-like structure in a fixture block. 

Might change slightly when connecting it with case search (https://github.com/dimagi/commcare-hq/pull/10981) at a later time. 
Adding in a different PR, since that one is big enough already, and this works as a separate change anyway.

FYI: @kaapstorm @czue 
cc: @orangejenny @gcapalbo

### CODE REVIEW

**Comment by kaapstorm** [comment]
Nice. (I don't watch Game Of Thrones, but I really like your tests. To make test writing and reading more enjoyable, I think they should have a back story.)

File: corehq/ex-submodules/casexml/apps/case/fixtures.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    def __init__(self, cases):
+        if not isinstance(cases, list):
+            self.cases = [cases]
+        else:
+            self.cases = cases
+
+    @property
+    def fixture(self):
+        """For a list of cases, return a fixture with all case properties
```
Comment (czue) [inline corehq/ex-submodules/casexml/apps/case/fixtures.py]: may be good to add link to the casedb spec

File: corehq/ex-submodules/casexml/apps/case/tests/data/case_fixture.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<fixture id="case">
+    <case case_id="redwoman" case_type="priestess" owner_id="lordoflight" status="open">
```
Comment (czue) [inline corehq/ex-submodules/casexml/apps/case/tests/data/case_fixture.xml]: :dancer: :fire:

**Comment by czue** [comment]
looks good. tests are broken, but :+1: to merge if you ran the new tests

**Comment by orangejenny** [comment]
looks good to me

**Comment by proteusvacuum** [comment]
Self merging, since :+1: and :white_check_mark:

**Comment by proteusvacuum** [comment]
self merging, since :+1: and :white_check_mark:

**Pull request merged**